### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by coliff**
and fix a few URLs

### CODE REVIEW

**Comment by bep** [comment]
All of these looks good, but could you resolve the merge conflict (as in: rebase against master)? I have previously tested the "Resolve conflicts" button with limited success.

**Comment by coliff** [comment]
Ok good to go now

File: hugolib/shortcode.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ type shortcode struct {
 	// Before Hug0 0.55 we didn't send any shortcode output to the markup
 	// renderer, and this flag told Hugo to process the {{ .Inner }} content
 	// separately.
-	// The old behaviour can be had by starting your shortcode template with:
+	// The old behavior can be had by starting your shortcode template with:
```
Comment (bep) [inline hugolib/shortcode.go:209]: I know I have merged a similar "typo fix" before, but as these comments are mostly written by me, and I write British English, I don't think we can call the above a "typo". It's listed first in the Oxford dictionary (even as the first option):

https://www.oed.com/search/dictionary/?scope=Entries&q=behaviour

File: hugolib/shortcode.go [hunk 2]
```diff
@@ -422,7 +422,7 @@ func doRenderShortcode(
 			}
 		}
 
-		// Pre Hugo 0.55 this was the behaviour even for the outer-most
+		// Pre Hugo 0.55 this was the behavior even for the outer-most
 		// shortcode.
```
Comment (bep) [inline hugolib/shortcode.go:426]: See above.

File: resources/transform_test.go [hunk 3]
```diff
@@ -185,7 +185,7 @@ func TestTransform(t *testing.T) {
 
 		spec := newTestResourceSpec(specDescriptor{c: c})
 
-		// Two transformations with same id, different behaviour.
+		// Two transformations with same id, different behavior.
```
Comment (bep) [inline resources/transform_test.go:188]: See above.

**Review by bep — CHANGES_REQUESTED** [review]

File: hugolib/shortcode.go (hunk 1 above)
Comment (coliff) [reply hugolib/shortcode.go:209]: We could revert that if you prefer, but most English technical documentation is written as American English. I thought it’d be best for it to be consistent.

Comment (bep) [reply hugolib/shortcode.go:209]: I think you would find that most English technical documentation is written in the language of the people who created it. But OK, I guess one could argue that _some_ of the comments in this repo is also used in the HugoDocs, which we have said should be in Simple English.

**Pull request merged**